docs/yaml.md

```diff
@@ -181,6 +181,50 @@ bind mounted into a container.
 There are experimental `userns`, `uidMappings` and `gidMappings` options for user namespaces but these are not yet supported, and may have
 permissions issues in use.
 
+In addition to the parts of the specification above used to generate the OCI spec, there is a `runtime` section in the image specification
+which specifies some actions to take place when the container is being started.
+- `mkdir` takes a list of directories to create at runtime. These are created before the container is started, so they can be used to create
```

Comment:
Given @ijc comment on the other PR, it might be good to explicitly mention where this path is relative too (host fs or container fs)